core/src/com/biglybt/core/networkmanager/VirtualChannelSelector.java

```diff
@@ -422,14 +422,6 @@ public void destroy()
 
   public boolean isSafeSelectionModeEnabled() {  return SAFE_SELECTOR_MODE_ENABLED;  }
 
-  public void enableSafeSelectionMode() {
-    if( !SAFE_SELECTOR_MODE_ENABLED ) {
-      SAFE_SELECTOR_MODE_ENABLED = true;
-      COConfigurationManager.setParameter( "network.tcp.enable_safe_selector_mode", true );
-      initSafeMode();
```

Comment:
initSafeMode() is only called from the constructor:
```
    if( SAFE_SELECTOR_MODE_ENABLED ) {
      initSafeMode();
    }
```

There is also some if switches in the implementing methods like:
```
  public boolean isPaused( AbstractSelectableChannel channel ) {
	    if( SAFE_SELECTOR_MODE_ENABLED ) {
              ...
	    }
	    else {
	      return( selector_impl.isPaused( channel ));
	    }
  }
```

@parg: What do you suggest to be done with all this?

Reply:
If the code has become unreachable purely from following through things that are logical tautologies then go ahead and remove them. Just don't remove stuff based on vague comments or things that appear to be true but can't be proved as such :)

Reply:
Well served. Haven't heard that word in quite a while :)